solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpJdkSolrClient.java

```diff
@@ -73,7 +74,7 @@ public class HttpJdkSolrClient extends HttpSolrClientBase {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   private static final String USER_AGENT =
-      "Solr[" + MethodHandles.lookup().lookupClass().getName() + "] 1.0";
+      "Solr[" + MethodHandles.lookup().lookupClass().getName() + "] " + SolrVersion.LATEST_STRING;
```

Comment:
It's ok to lose the distinction in version number because we still have v1 and v2 clients in the class names?

Reply:
Yes.